src/group.h

```diff
@@ -62,6 +62,9 @@ struct GroupStatistics {
 	static void UpdateAutoreplace(CompanyID company);
 };
 
+#if defined(_WIN32)
+#undef GF_END
+#endif /* _WIN32 */
```

Comment:
This shouldn't be needed any more since `GF_END` in our code got renamed. This PR also conflicts on this file.